templates/shortcodes/product_overview.html

```diff
@@ -0,0 +1,20 @@
+<div class="product-display" style="padding: 0 0; display: flex; align-items: center;">
+	<div class="product-info">
+		<div class="product-text-block">
+			{% if subtitle %}
+				<div class="product-subtitle">{{ subtitle }}</div>
+			{% endif %}
+			{% if title %}
+				<h1 class="product-title">{{ title }}</h1>
+			{% endif %}
+			<div class="product-subtitle">
+				{{ body | markdown | safe }}
+			</div>
+		</div>
+	</div>
+	{% if img %}
+		<div class="product-image-container">
+				<img src="/{{ img }}" alt="{{ alt | default(value='image') }}"/>
```

Comment:
this can be `<img src="/{{ img }}" alt="{{ alt | default(value=title) }}"/>` if `title` is required.